src/session/content/item_row.rs

```diff
@@ -94,16 +96,290 @@ impl ItemRow {
+                                                        ))
+                                                    }
+                                                )
+                                            }
+                                        }
+                                    }
+                                ));
+                            }
+                            MessageContent::MessageChatDeleteMember(data) => {
+                                self.get_or_create_event_row().set_label(&format!(
+                                    "<b>{}</b>",
+                                    match message.sender() {
+                                        MessageSender::User(user) if user.id() == data.user_id => {
+                                            if message.is_outgoing() {
+                                                gettext("You left the group")
+                                            } else {
+                                                gettext!("{} left the group", sender_name)
+                                            }
+                                        }
+                                        _ => {
+                                            let user_name = self::sender_name(
+                                                &message
+                                                    .chat()
+                                                    .session()
+                                                    .user_list()
+                                                    .get(data.user_id),
+                                            );
+                                            if message.is_outgoing() {
+                                                gettext!("You removed {} from the group", user_name)
+                                            } else {
+                                                gettext!(
+                                                    "{} removed {} from the group",
+                                                    sender_name,
+                                                    user_name
+                                                )
+                                            }
+                                        }
+                                    }
+                                ));
                             }
-                            _ => self.set_child_row::<MessageText>(message),
+                            MessageContent::MessagePinMessage(data) => {
+                                let msg =
+                                    match message.chat().history().message_by_id(data.message_id) {
+                                        Some(data) => match data.content().0 {
+                                            MessageContent::MessageText(data) => {
+                                                let msg = data.text.text;
+                                                if msg.chars().count() > 21 {
```

Comment:
I forgot to mentioned, but I changed pinned message length to 21 chars. It's the same as in official android and ios client + it's more readable than 17 chars (telegram desktop)

Reply:
I'm ok with that. @melix99 ?

Reply:
That's fine for me too, but I'd prefer keeping this value in a constant. Something like `MESSAGE_TRUNCATED_LENGTH` could work.

Reply:
But do we want to use three constants for each module or declare it in one place and import when needed?

Reply:
I think it's better to declare it in utils and importing it when needed.